src/data/mv_polynomial/supported.lean

```diff
@@ -0,0 +1,97 @@
+/-
+Copyright (c) 2021 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import ring_theory.adjoin.polynomial
+import data.mv_polynomial.variables
+
+/-!
+# Polynomials supported by a set of variables
+
+This file contains the definition and lemmas about `mv_polynomial.supported`.
+
+## Main definitions
+
+* `mv_polynomial.supported` : Given a set `s : set σ`, `supported R s` is the subalgebra of
+  `mv_polynomial σ R` consisting of polynomials whose set of variables is contained in `s`.
```

Comment:
So it's isomorphic (as algebra) to `mv_polynomial s R`, right? I think this docstring should mention that.

Reply:
Done